docs/docs-content/clusters/data-center/nutanix/create-manage-nutanix-cluster.md

```diff
@@ -0,0 +1,128 @@
+---
+sidebar_label: "Create and Manage Nutanix Cluster"
+title: "Create and Manage Nutanix Cluster"
+description: "Learn how to create and manage Nutanix clusters in Palette."
+hide_table_of_contents: false
+sidebar_position: 25
+tags: ["data center", "nutanix"]
+---
+
+Palette supports creating and managing Kubernetes clusters deployed to a Nutanix infrastructure environment. This section guides you in creating a Kubernetes cluster in a Nutanix cloud managed by Palette.
+
+## Prerequisites
+
+- A Nutanix cloud account added to Palette. Refer to [Add Nutanix Cloud Account](add-nutanix-cloud-account.md).
+
+<!-- A Nutanix Prism Central cloud account. For more information, review [Add Nutanix Cloud Account](add-nutanix-cloud-account.md).  -->
+
+- A Nutanix Private Cloud Gateway (PCG) deployed. For guidance, review [Install Private Cloud Gateway](./install-pcg/install-pcg.md).
+
+- An infrastructure cluster profile created for the Nutanix cloud. For guidance on creating a profile, refer to [Create an Infrastructure Profile](../../../profiles/cluster-profiles/create-cluster-profiles/create-infrastructure-profile.md). At the **Cloud Type** step of profile creation, select **Nutanix** listed under the **Tech Preview**. Select the custom packs provided at the OS and Kubernetes layers. Palette provides out-of-the-box packs for the network and storage profile layers, including the **Nutanix CSI** storage pack.
+
+- A Nutanix Prism Central account with *Prism Admin* role.
+
+- A Nutanix Prism Element cluster created.
+
+- A Nutanix subnet created in Nutanix Prism Central that will be assigned to the virtual machines (VMs) deployed in the Kubernetes cluster.
```

Comment:
@caroldelwing VMs will not be deployed "in" the K8s cluster: they will "make up" (not sure if there is a better word?) the k8s cluster.

```suggestion
- A Nutanix subnet created in Nutanix Prism Central that will be assigned to the virtual machines (VMs) that will make up the Kubernetes cluster.
```